test/test_operations.py

```diff
@@ -1893,6 +1893,21 @@ def test_sigmoid_bounds(self):
       assert torch.all(lower_bound >= 0.0)
       assert torch.all(upper_bound <= 1.0)
 
+  def test_cached_addcdiv(self):
+    xla_device = xm.xla_device()
+    met.clear_all()
+
+    t1 = torch.randn(1, 3).to(xla_device)
+    t2 = torch.randn(1, 3).to(xla_device)
+    t3 = torch.randn(1, 3).to(xla_device)
+    t1.addcdiv_(t2, t3, value=0.1)
+    # The following two scalars shouldn't trigger TransferToServerTime.
+    t1.addcdiv_(t2, t3, value=0.1)
```

Comment:
Let's may this test a bit more explicit by checking the `met.metric_data("TransferToServerTime")[0]` before second `addcdiv_` and after second `addcdiv_` to verify that second `addcdiv_` does not introduce new transfer.

Reply:
Okay, let me do that.